src/libs/actions/Session/index.ts

```diff
@@ -871,6 +871,33 @@ function waitForUserSignIn(): Promise<boolean> {
     });
 }
 
+/**
+ * check if the route can be accessed by anonymous user
```

Comment:
Updating this comment (or adding a comment in the function) to say why an anonymous user can only access different routes would be helpful for future programmers like me who don't have context on why this exists (in general, comments should be used to explain _why_ the code is like that, not _what_ the code is obviously doing)